recipe/meta.yaml

```diff
@@ -26,10 +26,9 @@ requirements:
     - pamela >=1.0
     - pegasus-wms.api
     - pegasus-wms.common
-    - pegasus-wms.dax
```

Comment:
Was this meant to be dropped? Asking as there is [a related test failure on `main`]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=656457&view=logs&jobId=656edd35-690f-5c53-9ba3-09c10d0bea97&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d )

Reply:
No! This rightly failed the CI, I have no idea why I then merged it. New PR incoming.

Reply:
No worries. Thanks Duncan! 🙏

Fixed with PR ( https://github.com/conda-forge/pegasus-wms-feedstock/pull/6 )